### PR TITLE
update sophos doc

### PR DESCRIPTION
**Pull request by hitarth-sumo**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md [hunk 1]
```diff
@@ -14,8 +14,12 @@ The Sophos Central Source provides a secure endpoint to receive authentication l
 The Sophos Central Source ingests:
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md:15]:
```suggestion
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md [hunk 2]
```diff
@@ -14,8 +14,12 @@ The Sophos Central Source provides a secure endpoint to receive authentication l
...
 
  * Alerts from the Sophos [Alerts API Endpoint](https://developer.sophos.com/docs/common-v1/1/routes/alerts/get).
+    - Endpoint data is consumed for the alerts whose product value is of type `endpoint`.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md:17]:
```suggestion
    - Endpoint data is consumed for alerts with a product value of type `endpoint`.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md [hunk 3]
```diff
@@ -14,8 +14,12 @@ The Sophos Central Source provides a secure endpoint to receive authentication l
...
  * Events from the Sophos [Events API Endpoint](https://developer.sophos.com/docs/siem-v1/1/routes/events/get)
 
+:::note
+To link the endpoint data to the alert, users can map the `alert.ManagedAgent.ID` field from the alert response with the `endpointID` field from the endpoint response.
```
Comment (JainM6) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md:21]:
```suggestion
To link the endpoint data to the alert, you can map the `alert.ManagedAgent.ID` field from the alert response with the `endpointID` field from the endpoint response.
```

**Review by JainM6 — CHANGES_REQUESTED** [review]
Thank you for submitting the PR. I found only three minor changes.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md (hunk 2 above)
Comment (hitarth-sumo) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md]: Done

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md (hunk 1 above)
Comment (hitarth-sumo) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md]: Done

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md (hunk 3 above)
Comment (hitarth-sumo) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/sophos-central-source.md]: Done

**Comment by hitarth-sumo** [comment]
> Thank you for submitting the PR. I found only three minor changes.

Thanks for the review. Updated the PR with the changes.

**Review by JainM6 — APPROVED** [review]

**Pull request merged**